Makefile

```diff
@@ -567,6 +571,10 @@ ci-test:
 	$(GO) test ./... -race -test.count 25 -cover
 	$(GO) test ./... -run IgnoreRace$$ -cover
 
+stress-test-%: install-stress-test-tools
+	$(REPOSITORY_ROOT)/python/bin/locust -f $(REPOSITORY_ROOT)/test/stress/frontend.py --host=http://localhost:51504 \
```

Comment:
Bring back the old style python virtualenv that lives in the build/python/ directory.